src/simularium/types.ts

```diff
@@ -1,4 +1,4 @@
-import { GeometryDisplayType } from "./VisGeometry/types";
+import { GeometryDisplayType } from "../visGeometry/types";
```

Comment:
this is an example of a data structure that should be factored out of visGeometry and into a common module so that simularium doesn't have to depend on visGeometry.